plat/common/include/pci/pci_bus.h

```diff
@@ -207,8 +207,8 @@ static struct pci_bus_handler ph __unused;
 #define PCI_ENABLE_BIT              (1u << 31)
 
 #define PCI_CONF_CLASS_ID          (0x08)
-#define PCI_CONF_CLASS_ID_SHFT     (16)
-#define PCI_CONF_CLASS_ID_MASK     (0xFF00)
+#define PCI_CONF_CLASS_ID_SHFT     (24)
```

Comment:
AFAIK the `Class Code` is 24-bits in size, but the shift from offset `0x08` is actually `8`, right?